domain/src/main/java/camp/nextstep/edu/github/domain/GetRepositoriesUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package camp.nextstep.edu.github.domain
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import javax.inject.Inject
+
+class GetRepositoriesUseCase @Inject constructor(
+    private val githubRepository: GithubRepository
+) {
+    suspend operator fun invoke(): Flow<GithubRepositories> {
```

Comment:
```suggestion
   operator fun invoke(): Flow<GithubRepositories> {
```
flow는 흐르는 의미로써, suspend의 일시정지의 의미가 아니에요.
suspend 키워드를 제거해보면 어떨까요?